src/pages/Cheon/Main/Feed.js

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from 'react';
+
+const Feed = props => {
+  const { feed } = props;
```

Comment:
저도 아직 적용을 안해 봤는데 이경우 아래오 같이 쓸 수 있다고 멘토님께 리뷰 받았습니다.
```suggestion
const Feed = ({feed}) => {
```